src/dashboard.tsx

```diff
@@ -296,6 +298,14 @@ export class URLInput extends Widget {
     return this._urlChanged;
   }
 
+  /**
+   * The in browser dashboard check for authenticated dashboards.
+   */
+  set browserDashboardCheck(value: boolean) {
+    this.browserDashboardCheck = value;
+    this.update();
```

Comment:
I don't think this `update()` is necessary (though it doesn't do much harm either).